scripts/check_python_lint.sh

```diff
@@ -38,19 +39,22 @@ is_test() {
 }
 
 TEST_ARGS="${ARGS} $(emit_test_nolint)"
-while read -d '' filename; do
+while read -r -d '' filename; do
   if is_test "${filename}"
   then
     args="${TEST_ARGS}"
   else
     args="${ARGS}"
   fi
-  if ! ${LINT} ${args} "${filename}"
+  if ! ${LINT} "${args}" "${filename}"
```

Comment:
This is treating `args` as a single argument. You would need to use an array to split it space safely. Or just remove the quotes. I don't see any clear way a space could get into an argument.